types.ts

```diff
@@ -318,9 +326,7 @@ export type ValueSpecUnion = {
   /** What tag for the specification, for tag unions */
   tag: {
     id: string;
-    /** @deprecated - optional only for backwards compatibility */
-    name?: string;
-    /** @deprecated - optional only for backwards compatibility */
+    name: string;
```

Comment:
Isn't this also optional?

Reply:
No, it is always required here on the tag.

Reply:
New FE [types](https://github.com/Start9Labs/embassy-os/pull/2128/files#diff-8ac7accad8e366cf57c222addbf7fa2018c7083c3ccf4fa106c799746646536dR56-R168)